frameworks/cassandra/universe/resource.json

```diff
@@ -7,7 +7,7 @@
       "cassandra-openssl-tar-gz": "https://downloads.mesosphere.com/cassandra/assets/openssl-1.0.2n-libs.tar.gz",
       "libmesos-bundle-tar-gz": "{{libmesos-bundle-url}}",
       "scheduler-zip": "{{artifact-dir}}/cassandra-scheduler.zip",
-      "cassandra-tar-gz": "https://downloads.mesosphere.com/cassandra/assets/apache-cassandra-3.0.16-bin-dcos.tar.gz",
+      "cassandra-tar-gz": "https://downloads.mesosphere.com/cassandra/assets/apache-cassandra-3.11.3-bin.tar.gz",
```

Comment:
is this straight from apache cassandra downloads or did you have to modify the tarball with dcos bits? I ask bc all previous versions in the `assets/` folder includes the `-dcos` suffix. If so, pls rename and reupload